### PR TITLE
test: minor fixes in RQTT [MINOR]

### PR DESCRIPTION
**Pull request by big-andy-coates**
### Description 

Fixes a few minor issues in QTT/RQTT.

### CODE REVIEW

File: ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/loader/JsonTestLoader.java [hunk 1]
```diff
@@ -146,10 +145,8 @@ private static void throwOnDuplicateNames(final List<? extends Test> testCases)
         .stream()
         .filter(e -> e.getValue().size() > 1)
         .map(e -> "test name: '" + e.getKey()
-            + "' found in files: " + e.getValue().stream()
-            .map(Test::getTestLocation)
-            .map(TestLocation::toString)
-            .collect(Collectors.joining(",")))
```
Comment (big-andy-coates) [inline ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/loader/JsonTestLoader.java:152]: This was pointless as `getTestLocation` finds the first test in the file with the supplied name - so all get the same location.

File: ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/rest/RestTestExecutor.java [hunk 2]
```diff
@@ -215,14 +217,19 @@ private void produceInputs(final Map<String, List<Record>> inputs) {
...
           topicInfo.getValueSerializer()
       )) {
-        for (final Record record : records) {
-          producer.send(new ProducerRecord<>(
-              topicName,
-              null,
-              record.timestamp().orElse(0L),
-              record.key(),
-              record.value()
-          ));
```
Comment (big-andy-coates) [inline ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/rest/RestTestExecutor.java:225]: This code was neither waiting for the message to actually be sent, not failing if the message failed to send.

Comment (vcrfxia) [reply ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/rest/RestTestExecutor.java:225]: How did the tests pass if the test wasn't waiting for the records to be sent? Was the downstream validation waiting for records to arrive? (Took a quick skim of the code but didn't see it.) Was it just luck that record production tended to be quick?

**Review by vcrfxia — APPROVED** [review]
LGTM. One question (mostly of curiosity) inline.

File: ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/rest/RestTestExecutor.java (hunk 2 above)
Comment (big-andy-coates) [reply ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/rest/RestTestExecutor.java:225]: I think there's code that waits for the response for a period of time.  It's mainly about detecting failures.

**Pull request merged**